packages/scripts/src/service/parameterService.ts

```diff
@@ -1,5 +1,5 @@
 const freeMarkerRegEx = /\${([a-zA-Z0-9-#]+)}/g // Match foo from ${foo}
-const magmaRegEx = /\$\(\"([a-zA-Z0-9-#]+)\"\)|$\(\'([a-zA-Z0-9-#]+)\'\)/g // Match foo from $('foo') or $("foo")
+const magmaRegEx = /\$\(\"([a-zA-Z0-9-#]+)\"\)|\$\(\'([a-zA-Z0-9-#]+)\'\)/g // Match foo from $('foo') or $("foo")
```

Comment:
please add a test case

Reply:
I've fixed the broken test that was supposed to test single quotes